packages/data-service-generator/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "data-service-generator",
-  "version": "1.9.10",
+  "version": "1.9.11",
```

Comment:
It was failing on version test so I had to change it. 🤔

Reply:
@Shurtu-gal please revert this change since in the new release flow an automated workflow will increment this version and it doesn't need to be done manually anymore.

Reply:
@Shurtu-gal, we (@overbit ) changed this workflow, so it should no longer fail on this. If you pull from our repo into your fork, you should also not see failures in the CI anymore.
@overbit, please correct me if I'm misleading Ashish here.

Please just revert this single change, and we can approve the PR. Great work!

Reply:
@mulygottlieb I have reverted the version.